.gitignore

```diff
@@ -93,8 +93,9 @@ $RECYCLE.BIN/
 # Source maps
 *.map
 
-# We don't need to keep the Swagger UI code under revision control
-docs/*
+# Schemas are autogenerated to be includes in OpenAPI Specification
+schemas
+
+# Generated folder for static hosting
+dist
```

Comment:
We should only keep our project-specific code under revision control. Any auto-generated files or static dependencies should be included from external sources, when possible.